hack/setup-ci-test.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
```

Comment:
please set those bash options at the beginning
```

set -o nounset
set -o errexit
set -o pipefail
```

Reply:
Good point, done :heavy_check_mark: